examples/grpc/helloworld_client/main.go

```diff
@@ -37,8 +37,8 @@ func main() {
 	// the collected data.
 	view.RegisterExporter(&exporter.PrintExporter{})
 
-	// Register the view to collect client request count.
-	if err := view.Register(ocgrpc.ClientErrorCountView); err != nil {
```

Comment:
I have a naive question, how are now going to be able to track the number of errors in either the client or server?

Reply:
We now break down the requests by status, so you can get the total errors by adding up all the non-success statuses like we do in rpcz.